src/gui/mzroll/isotopeplot.cpp

```diff
@@ -63,13 +63,17 @@ void IsotopePlot::replot()
 void IsotopePlot::setPeakGroup(PeakGroup* group) {
     //cerr << "IsotopePlot::setPeakGroup()" << group << endl;
     if ( group == NULL ) return;
+
+    if (!group->tableName().empty())
+        group->childrenBarPlot = group->children;
     if (group->childCountBarPlot() == 0) return;
 
     if (group->isIsotope() && group->getParent() ) {
         setPeakGroup(group->getParent());
         return;
     }
 
+    emit peakGroupSet(!group->tableName().empty());
```

Comment:
Have we disabled the labels after plotting from isotope plot, on purpose?

Reply:
Yes. So that the user may not get the idea that they can change the plots for integrated groups (basically any group from that is enlisted in a peak-table).

Reply:
Okkayy..